src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -5,7 +5,6 @@
 using System.Reflection;
 using System.Collections;
 
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2112:SecuredTypesShouldNotExposeFields", Scope = "type", Target = "System.ComponentModel.AttributeCollection")]
```

Comment:
Please don't remove FXCop suppressions -- those affect our ability to sign off on the code.